README.md

```diff
@@ -7,6 +7,68 @@ bells and whistles.
 For a longer description and available features and options, see the
 [man page](mkosi.md).
 
+# Installation
+
+Installing `mkosi` is easy, as it has no runtime Python dependencies (you will
+need all the tools to format filesystems and bootstrap the distribution
+appropriate for your image, though).
+
+If you just want the current master branch you can run
+```shell
+python3 -m pip install --user git+https://github.com/systemd/mkosi.git
+```
+
+If you want to hack on mkosi do
+```shell
+# clone either this repository or your fork of it
+git clone https://github.com/systemd/mkosi/
+cd mkosi
+python3 -m pip install --user --editable .
+```
+This will install mkosi in editable mode to `~/.local/bin/mkosi`, allowing you
+to use your own changes right away.
+
+For development you optionally also need [mypy](https://github.com/python/mypy)
+and [pytest](https://github.com/pytest-dev/pytest). We check tests and typing in
+CI (see `.github/workflows`), but you can run the tests locally as well.
+
+## zipapp
+
+You can also packup mkosi as a
+[zipapp](https://docs.python.org/3/library/zipapp.html) that you can deploy
+anywhere in your `PATH`. This will leave an `mkosi` binary in `outdir/`
+```shell
+mkdir -p builddir outdir
```

Comment:
Yeah, we can put this in a scripts directory. Maybe put do_a_release in there as well

Reply:
Make sure to clean up the build directory as well after generating the zipapp